indexer/e2e_tests/bridge_transfers_e2e_test.go

```diff
@@ -445,3 +447,76 @@ func TestE2EBridgeTransfersCursoredWithdrawals(t *testing.T) {
 		require.Equal(t, int64(3-i)*params.Ether, withdrawal.L2BridgeWithdrawal.Tx.Amount.Int64())
 	}
 }
+
+func Test_ClientGetWithdrawals(t *testing.T) {
```

Comment:
nice!

Reply:
Bump here about snake case with the test name